src/Lib/Managers/SelectedBoundManager.vala

```diff
@@ -353,15 +369,51 @@ public class Akira.Lib.Managers.SelectedBoundManager : Object {
         // Increment the cairo matrix coordinates so we can ignore the item's rotation.
         matrix.x0 += first_move_x;
         matrix.y0 += first_move_y;
-
         item.set_transform (matrix);
 
-        // Any adjustments like snapping would be computed here, and the item would
-        // then be translated again.
+        // Make adjustment basted on snaps
+        // Double the sensitivity to allow for reuse of grid after snap
+        var sensitivity = (int) Utils.Snapping.adjusted_sensitivity (canvas.current_scale);
```

Comment:
If we always need an INTEGER from this method, we should return directly an integer from the method instead of type casting it every time we use it.